lib/features/digital_guide_view/presentation/digital_guide_view.dart

```diff
@@ -0,0 +1,121 @@
+import "package:auto_route/annotations.dart";
+import "package:fast_immutable_collections/fast_immutable_collections.dart";
+import "package:flutter/material.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+
+import "../../../config/ui_config.dart";
+import "../../../gen/assets.gen.dart";
+import "../../../utils/context_extensions.dart";
+import "../../../utils/determine_contact_icon.dart";
+import "../../../widgets/detail_views/contact_section.dart";
+import "../../../widgets/detail_views/detail_view_app_bar.dart";
+import "../../../widgets/my_cached_image.dart";
+import "../../../widgets/my_error_widget.dart";
+import "../../../widgets/report_change_button.dart";
+import "../data/models/digital_guide_response.dart";
+import "../data/repository/digital_guide_repository.dart";
+import "../data/repository/image_repository.dart";
+import "widgets/accessibility_button.dart";
+import "widgets/digital_guide_data_source_link.dart";
+import "widgets/digital_guide_features_section.dart";
+import "widgets/headlines_section.dart";
+
+@RoutePage()
+class DigitalGuideView extends ConsumerWidget {
+  const DigitalGuideView({
+    @PathParam("id") required this.id,
+  });
+
+  final int id;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final asyncDigitalGuideData = ref.watch(getDigitalGuideDataProvider(id));
+    // question: Should the app bar appear during loading or when there's an error?
+    // Now it doesn't, neither does it appear on SKS menu screen
+    return asyncDigitalGuideData.when(
+      data: _DigitalGuideView.new,
+      error: (error, stackTrace) => MyErrorWidget(error),
+      loading: () => const Scaffold(
+        body: Center(
+          child: CircularProgressIndicator(),
+        ),
+      ),
+    );
+  }
+}
+
+class _DigitalGuideView extends ConsumerWidget {
+  const _DigitalGuideView(this.digitalGuideResponse);
+
+  final DigitalGuideResponse digitalGuideResponse;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final asyncImageResponse =
+        ref.watch(getImageDataProvider(digitalGuideResponse.images[0]));
```

Comment:
tbh we will wanna show our photos from our backend here, but it's outside this tasks scope. Once we connect it properly to our buildings, we'll implement this. But since you're already making a lot of todo comments, you can leave one here too

Reply:
Do we want to mix two backends? 
I would love You look in next commit at digital_guide_repository as imageUrl retrieval will be changed

Reply:
it can stay for now 100%, but in the future yeah, we'll mix them anyway